waspc/packages/deploy/src/providers/fly/helpers/tomlFileHelpers.ts

```diff
@@ -56,18 +56,30 @@ export function getAppNameFromToml(path: string): string {
 	return data.app;
 }
 
-export function getInferredBasenameFromServerToml(paths: TomlFilePaths): string {
+export function getInferredBasenameFromServerToml(
+	paths: TomlFilePaths,
+): string {
 	const serverName = getAppNameFromToml(paths.serverTomlPath);
 	return serverName.replace('-server', '');
 }
 
-export function getInferredBasenameFromClientToml(paths: TomlFilePaths): string {
+export function getInferredBasenameFromClientToml(
+	paths: TomlFilePaths,
+): string {
 	const clientName = getAppNameFromToml(paths.clientTomlPath);
 	return clientName.replace('-client', '');
 }
 
-export function replaceLineInLocalToml(searchValue: string | RegExp, replaceValue: string): void {
+export function replaceLineInLocalToml(
+	searchValue: string | RegExp,
+	replaceValue: string,
+): void {
 	const content = fs.readFileSync('fly.toml', 'utf8');
 	const updatedContent = content.replace(searchValue, replaceValue);
 	fs.writeFileSync('fly.toml', updatedContent);
 }
+
```

Comment:
Instead of implementing a separate function, I would probably instead have `replaceLineInLocalToml` return or throw an error, and would then handle that and ask a question based on that. But ok, this also works (we just read the file twice and have a bit of duplication here).